google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -482,6 +504,7 @@ public boolean equals(Object o) {
         && Objects.equals(priority(), that.priority())
         && Objects.equals(tag(), that.tag())
         && Objects.equals(etag(), that.etag())
+        && Objects.equals(dataBoostEnabled(), that.dataBoostEnabled())
         && Objects.equals(validateOnly(), that.validateOnly())
         && Objects.equals(withOptimisticLock(), that.withOptimisticLock());
```

Comment:
```suggestion
        && Objects.equals(validateOnly(), that.validateOnly())
        && Objects.equals(withOptimisticLock(), that.withOptimisticLock())
        && Objects.equals(dataBoostEnabled(), that.dataBoostEnabled());
```
nit: let's keep the ordering same.